neo-cli/CLI/Logger.cs

```diff
@@ -0,0 +1,100 @@
+using Neo.Plugins;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using static System.IO.Path;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+using Neo.ConsoleService;
```

Comment:
```suggestion
using Neo.Plugins;
using System;
using System.Text;
using System.IO;
using static System.IO.Path;
using System.Reflection;
using Neo.ConsoleService;
```
Delete unused dependency.